### PR TITLE
Bug: DB setup

### PR DESCRIPTION
**Pull request by doncem**
### Change description ###

Extract db setup related changes from main infrastructure code stack of cherry picks - next PR

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,17 +8,15 @@ services:
...
         - https_proxy
         - no_proxy
-    command: --wait-for-database 30
     image: docker.artifactory.reform.hmcts.net/reform/job-scheduler-api
     container_name: job-scheduler-api
     environment:
       - S2S_URL
       - S2S_SECRET
-      - S2S_NAME
-      - S2S_TTL
```
Comment (rkondratowicz) [inline docker-compose.yml]: why were those removed??

Comment (doncem) [reply docker-compose.yml]: perhaps because not used? was within same cherry-pick really

Comment (rkondratowicz) [reply docker-compose.yml]: nothing to do with DB

Comment (rkondratowicz) [reply docker-compose.yml]: Are you sure? If secret is needed then so is name.
Also, let's keep this PR about 'Bug: DB setup'

**Review by rkondratowicz — APPROVED** [review]
